client/my-sites/email/email-providers-comparison/stacked/index.tsx

```diff
@@ -86,12 +99,53 @@ const EmailProvidersStackedComparison = ( {
 		)
 	);
 
+	useQuerySitePurchases( selectedSite?.ID ?? -1 );
+
+	const currentUserCanAddEmail = canCurrentUserAddEmail( domain );
+
+	const isPrivacyAvailable = domain?.privacyAvailable;
+
+	const contactOwnerUrl = `https://privatewho.is/?s=${ selectedDomainName }`;
+
+	const purchses = useSelector( ( state ) => getSitePurchases( state, selectedSite?.ID ) );
+
+	const domainSubscription = purchses.filter(
+		( purchase ) => purchase.id === parseInt( domain?.subscriptionId ?? 0 )
+	)[ 0 ];
+
+	const fetchOptions = {
+		search: domainSubscription?.userId,
+		search_columns: [ 'ID' ],
+	};
+
+	const { data } = useUsersQuery( selectedSite?.ID, fetchOptions, {
+		enabled: domainSubscription !== undefined,
+	} );
+
+	const teams = data as InfiniteData< UsersData > & UsersData;
+
+	const ownerUserName = teams ? teams.users[ 0 ].login : '';
```

Comment:
Since we'll need similar logic for the Email Management page as well to surface messaging to non-email-subscription owners, it's probably worthwhile to create a helper function for getting the domain owner username? It'll also improve modularity of the code.